src/Trace/Tracer/ExtensionTracer.php

```diff
@@ -55,9 +55,9 @@ class ExtensionTracer implements TracerInterface, SpanEventHandlerInterface
      * Create a new ExtensionTracer
      *
      * @param SpanContext|null $initialContext The starting span context.
-     * @param null $exporter.
+     * @param null $exporter
      * @param array $options
-     *      @type int span_buffer_limit, overrides the default span buffer limit
+     * @type int span_buffer_limit, overrides the default span buffer limit
```

Comment:
$options has `span_buffer_limit`
```suggestion
     * - @type int span_buffer_limit, overrides the default span buffer limit
```